test-suite/coq-makefile/template/path-init-tools.sh

```diff
@@ -0,0 +1,4 @@
+set -e
+set -o pipefail
```

Comment:
Why do you need both `set` invocations?

Reply:
Actually this is dead code, I'll remove the file.